README.md

```diff
@@ -66,7 +66,21 @@ file.csslint.results = []; // CSSLint errors
 file.csslint.opt = {}; // The options you passed to CSSLint
 ```
 
-## Custom Reporters
+## Using reporters
+
+Using built-in reporters are quite easy. First pipe your css-files in `gulp-csslint`, then pipe them through `csslint.reporter`, specifying which reporter you'd like to use.
```

Comment:
Alternate wording:

Several reporters come built-in to css-lint. To use one of these reporters, pass the name to `csslint.reporter`, along with an optional file path.